src/Adapter/Customer/CommandHandler/EditCustomerHandler.php

```diff
@@ -215,6 +215,12 @@ private function assertCustomerWithUpdatedEmailDoesNotExist(Customer $customer,
             return;
         }
 
+        // We check this only if dealing with a registered customer
+        // Two guests with the same email can co-exist, two registered customers can not
+        if ($customer->is_guest == 1) {
```

Comment:
```suggestion
        if ($customer->is_guest === 1) {
```

Doesn't PHP have strict operators? Is it a good practice?

Reply:
@NeOMakinG updated